lib/util/index.ts

```diff
@@ -252,25 +223,3 @@ const convertExpandToObject = <T extends {}>(
 
 	return cloneDeep(expandOption);
 };
-
-// In order not to introduce a breaking change, we export each element independently and all together as a default export.
-export default {
```

Comment:
We no longer re-export everything in a default export :tada: 
Now that nothing is in coffee, it was clear that we can still use the individual exports with `require()` calls.